k8s/environments/sbox/common-overlay/mi/kustomization.yaml

```diff
@@ -1,8 +1,18 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-- ../../../../namespaces/mi
+- ../../../../namespaces/mi/mi-developer-rbac.yaml
```

Comment:
are these files to be migrated in a follow up PR?

Reply:
yes